.github/workflows/docker-image.yml

```
@@ -0,0 +1,98 @@
+name: Docker
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+  push:
+    branches: [ "main", "**" ]
```

```suggestion
    branches: [ "**" ]
```

Includes `main` already.